sources/RuleEditWindow.cpp

```diff
@@ -108,8 +114,8 @@ RuleEditWindow::MessageReceived(BMessage* msg)
 		{
 			if (strlen(fDescriptionBox->Text()) < 1) {
 				BAlert* alert = new BAlert("Filer",
-					"You need to add a description if you want to add this "
-					"rule to the list.", "OK");
+					B_TRANSLATE("You need to add a description if you want to "
+					"add this rule to the list."), B_TRANSLATE("OK"));
```

Comment:
Let's shorten that to "The rule needs to have a description."
Or something similar.